install.sh

```diff
@@ -15,10 +15,14 @@ TARGET=${TARGET-linux}
 libversion=3
 libVFile="./libversion"
 
+function has_cmd {
+	command -v "$1" >/dev/null 2>&1
```

Comment:
Why? `command -v` was also used before in the script and I thought that it was best practice.

see e.g. http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script